### ISSUE TITLE
the loss is higher after few epochs when I train on my own dataset

### ISSUE THREAD

**Issue by bing1zhi2** [davidsandberg/facenet#924]
Training on Asian training dataset . https://github.com/deepinsight/insightface/issues/256
the loss is lower at first ,but go higher after few epochs.
command:
 --logs_base_dir F:/work/runspace/log/  --models_base_dir F:/dataset/facenet/model/my/  --data_dir G:/dataset/train_celebrity/celebrity531_mtcnnpy_182  --image_size 160 --lfw_dir F:/dataset/lfw/lfw_mtcnnpy_160 --model_def models.inception_resnet_v1    --optimizer ADAM  --learning_rate -1  --max_nrof_epochs 1  --batch_size 50  --keep_probability 0.4  --random_flip  --use_fixed_image_standardization  --learning_rate_schedule_file data/learning_rate_schedule_classifier_vggface2.txt  --weight_decay 5e-4  --embedding_size 512  --lfw_distance_metric 1  --lfw_use_flipped_images  --lfw_subtract_mean  --validation_set_split_ratio 0.01  --validate_every_n_epochs 1 --gpu_memory_fraction 0.6 --pretrained_model F:/dataset/facenet/model/my/20181127-052123/model-20181127-052123.ckpt-1

Epoch: [1][79/1000]	Time 1.860	Loss 0.602	Xent 0.037	RegLoss 0.565	Accuracy 1.000	Lr 0.05000	Cl 0.598
Epoch: [1][80/1000]	Time 1.809	Loss 0.702	Xent 0.137	RegLoss 0.565	Accuracy 0.980	Lr 0.05000	Cl 0.674
Epoch: [1][81/1000]	Time 1.855	Loss 0.590	Xent 0.026	RegLoss 0.565	Accuracy 1.000	Lr 0.05000	Cl 0.675

then：
Epoch: [1][998/1000]	Time 1.330	Loss 2.124	Xent 1.124	RegLoss 1.000	Accuracy 0.720	Lr 0.05000	Cl 0.624
Epoch: [1][999/1000]	Time 1.227	Loss 1.657	Xent 0.656	RegLoss 1.000	Accuracy 0.780	Lr 0.05000	Cl 0.597
Epoch: [1][1000/1000]	Time 1.344	Loss 1.932	Xent 0.931	RegLoss 1.001	Accuracy 0.760	Lr 0.05000	Cl 0.593

**Comment by bing1zhi2** [comment]
I trained all the vars in the model before,but now I only train the last 'Logits' layer.The acc is good on flw dataset this time

**Comment by cp0000** [comment]
> I trained all the vars in the model before,but now I only train the last 'Logits' layer.The acc is good on flw dataset this time

hi,could u please share more details about how you only train the last 'Logits' layer.,thanks a lot
